examples/huggingface_examples/herg/hERG_inhib.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+get_ipython().system(' pip show PyTDC')
+
+
+RANDOM_SEED = 42
+
+
+import pandas as pd
+
+from tdc import utils as tdc_utils
+from tdc.single_pred import Tox
+
+from DeepPurpose import utils as dp_utils, CompoundPred
+
+
+# ## Prepare Data
+
+X, y = Tox(name = 'herg_central', label_name="hERG_inhib").get_data(format = 'DeepPurpose')
+
+
+X, y = X[:100], y[:100]
```

Comment:
This is to quickly iterate over the code during development. This line will be removed while tuning the final model.

Reply:
Sounds good!